rest/filters.go

```diff
@@ -0,0 +1,43 @@
+package rest
+
+import (
+	"github.com/Peripli/service-manager/pkg/filter"
+	"github.com/gobwas/glob"
+	"github.com/sirupsen/logrus"
+)
+
+// MatchFilters returns all filters that mach the provided endpoint's path and method
+func MatchFilters(endpoint *Endpoint, filters []filter.Filter) []filter.Filter {
+	matches := []filter.Filter{}
+	for _, filter := range filters {
+		if matchPath(endpoint.Path, filter.PathPattern) &&
+			matchMethod(endpoint.Method, filter.Methods) {
+			matches = append(matches, filter)
+		}
+	}
+	logrus.Debugf("%d filters for endpoint %v", len(matches), endpoint)
```

Comment:
Count is not very useful here. The filter names would be more useful for debugging

Reply:
Currently we don't have any naming on filters. Should we add names or identifiers?

Reply:
Added as an issue

Reply:
ok, added filter name